updated expectation management on sec issues (#437)
apache/cloudstack-www: src/pages/security.md

vulnerability, how it might be exploited, and any additional information that
might be useful.

Upon notification, the ASF security team will work with the CloudStack
PMC through validation and fixing the issue. If the issue is
validated, it will still take time to fix the issue. The amount of
time depends on the availability of volunteers and number people

> involved that have a stake in the issue. In later years it has turned

Reviewer: It might make sense to mention that the round-trip time also depends on the severity of the issue? Hopefully severe issues won't be open for six months ;)

> out to take up to six months, from notification to public announcement

Reviewer 2: do we need to be specific here? I don't think we need to mention this whole sentence at all

> out to take up to six months, from notification to public announcement
> of the vulnerability, due to parallel work on multiple issues. During

Author: ```suggestion
out to take up more and more time from notification to public announcement
of the vulnerability, due to parallel work on multiple issues. During
```

@borisstoyanov @raboof ?
